sources/packages/backend/apps/api/src/route-controllers/student-scholastic-standings/student-scholastic-standings.controller.service.ts

```diff
@@ -57,6 +60,29 @@ export class ScholasticStandingControllerService {
     };
   }
 
+  /**
+   * Get Scholastic Standing summary details.
+   * @param studentId student id to retrieve the scholastic standing summary.
+   * @returns Scholastic Standing summary details.
+   */
+  async getScholasticStandingSummary(options?: {
+    studentId: number;
+  }): Promise<ScholasticStandingSummaryDetailsAPIOutDTO> {
+    const scholasticStandingSummary =
+      await this.studentScholasticStandingsService.getScholasticStandingSummary(
+        options?.studentId,
+      );
+    const SFASUnsuccessfulCompletionWeeks =
```

Comment:
The local variables should start in lower case, in case it would be `sfasUnsuccessfulCompletionWeeks`.